.github/workflows/run-upstream-tests.yml

```diff
@@ -1,75 +0,0 @@
-name: run-upstream-tests
```

Comment:
This is not good. We need to modify ci-mgmt not to wipe away files it does not understand. This is a natural customization point.